### DISCUSSION TITLE
in Progress...

### DISCUSSION THREAD

**Discussion by phlnil** [blakeblackshear/frigate#12424] [General]
Hi ,
I am trying to tuning the system and discover something new everytime, 
about "Events" 
clicking on a Camera then Event:
I see "past events" and dozens of events 
but sometimes in the same session (not restarting Frigate or touching anything)
clicking on Events (left side main menu) I get a new "event" at the top of the other ones:
"Outgoing events" 10h ago... (In Progress)

what it is happening ? (cpu system is not under any stress... so this process is for the tpu alone?)

I am reading the thread [1] but was not clear how to process the debugging mode for that (In Progress) event.
(NickM-27 was referring just to a live debugging maybe)
[1]:https://github.com/blakeblackshear/frigate/issues/4583#issuecomment-1336183206
maybe i nee**d** to add _bounding_box: true_.

Is Frigate really yet processing a capture of 10 hours ago ? or what I can do ? delete it ? 
no relevant error on log.

Thank you.

**Comment by hawkeye217** [comment]
If Frigate detects an object and it doesn't move for 10 hours, then yes, this will be listed as an "ongoing event".

You can use Frigate's debug view to see what is being detected. In 0.13, click the "Cameras" link from your Frigate sidebar, click on your camera image, then click on "Debug" in the top right. Below the camera image is a gear with "Show options". Click that and turn on the "bounding box" switch.

**Reply by phlnil** [reply]
thank you for your reply.
yes, infact  all was on (except for timestamp)
but I needed to click on the little circle on the right play button, otherwise the clip (ongoing event) is  played without any bounding box...
(the clip is a collection of events (for 70sec) ona span of  2hours)
Probably is a stationary object that in the same position to create this issue.

I need to delete the clip ? it seems that frigate has done for me (creting a new one clip (now "2h ago" shows)
